lib/browser/guest-window-manager.js

```diff
@@ -47,8 +47,19 @@ const mergeBrowserWindowOptions = function (embedder, options) {
     options.webPreferences = {}
   }
   if (embedder.browserWindowOptions != null) {
+    let parentOptions = {...embedder.browserWindowOptions}
```

Comment:
/nit we'll be cloning the object needlessly 99% of the time here, maybe `let parentOptions = embedder.browserWindowOptions` and then spread it inside the `if` below